source/default_mode/DataNodes.h

```diff
@@ -66,6 +66,20 @@ emp::DataFile & SymWorld::SetupSymIntValFile(const std::string & filename) {
  */
 emp::DataFile & SymWorld::SetupHostIntValFile(const std::string & filename) {
   auto & file = SetupFile(filename);
+  AddHeadersToHostFile(file);
+  file.PrintHeaderKeys();
+  return file;
+}
+
+
+/**
+ * Input: The Empirical DataFile object tracking data nodes.
+ *
+ * Output: None.
+ *
+ * Purpose: To add host data nodes to be tracked to the host data file.

```

Comment:
Let's make this more specific/accurate. Probably should specifically mention that it sets up the columns for the host data file but specifically doesn't print the header